content/ru/docs/reference/glossary/control-plane.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Панель управления
```

Comment:
This seems like a translation of “control panel”. Is _Управления плоскость_ better? Same idea as “обслуживанием вызова и сигнализации плоскость”

(I don't speak Russian very well).

Reply:
@Potapy4  Я на самом деле сам не уверен в этом переводе, поскольку есть dashboard, который тоже переводится так же. Но с другой стороны он не так часто упоминается, и control panel как панель управления я видел в одной переводной книге. Плоскость управления встречал в другой книге, но странно немного звучит.. Какие есть мысли на этот счёт?

Reply:
You can touch a dashboard or a control panel, whereas a control plane is an imagined, logical concept. If the control plane were its own circuit board it would be sealed away, unseen.

The common English language counterparts to _control plane_ are “data plane” and “forwarding  plane”. Sometimes there is also a “management plane”.
https://www.juniper.net/documentation/en_US/day-one-books/HDN.pdf#page=337 has some diagrams.

Reply:
Thanks for the explanation.

@Potapy4 я думаю, лучше использовать тогда "плоскость", но в переводах других текстах используется "панель", поэтому я в отдельном PR сделаю соответствующие замены.

Reply:
@lex111 я тоже встречал плоскость - звучит странно, но раз уж так перевели, давай также